.github/workflows/test_selfhosted_runner.yml

```diff
@@ -0,0 +1,25 @@
+name: Test Github action on self hosted RPI runnes
+
+on:
+  push:
+    branches: [pull_request]
```

Comment:
Shouldn't this be
```yml
on:
  push:
    branches:
    - master
  pull_request:
```

or

```yml
on:
  pull_request:
```

Reply:
Now the test ran :) https://github.com/espressif/arduino-esp32/pull/5521/checks?check_run_id=3310070479

Could you sign the CLA too please? GH Checks are complaining

Reply:
@me-no-dev Done, thank you. PLS merge it.